ai_agents/agentic-form-filler/.env.example

```
@@ -0,0 +1,2 @@
+VISION_AGENT_API_KEY=your_landing_ai_key_here 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)